### ISSUE TITLE
Bug - [Description List] - [term help text popover not accessible by keyboard navigation]

### ISSUE THREAD

**Issue by rmscampos** [patternfly/patternfly-react#10846]
**Describe the problem**
A clear and concise description of the problem. Which components are affected?

https://www.patternfly.org/components/description-list/react/term-help-text/

Description list term help text popover can't be opened just by keyboard navigation alone. I have tried 'enter', 'space', and 'ctrl+option+space'. I think for accessibility reasons, this should be able to be interacted with via the keyboard. Screen reader functionality works as expected here, so this is just a basic keyboard navigation issue.

**How do you reproduce the problem?**
Provide steps to reproduce. 

https://www.patternfly.org/components/description-list/react/term-help-text/

Use keyboard (tabbing) to navigate to any of the term help texts like 'Name', 'Namespace', 'Labels', etc. Try to open the popover with only the keyboard. 

**Expected behavior**
The popover Description List text elements should be able to be interacted with via keyboard navigation

**Is this issue blocking you?**
No, but it ensuring accessibility compliance is part of the epic I am working on in the next few months.

**Screenshots**
If applicable, add screenshots to help explain the issue.

**What is your environment?**
 - OS: Sonoma 14.5
 - Browser: Chrome (this seems to be an issue across all browsers)

**What is your product and what release date are you targeting?**
Ecosystem Catalog
End of 2024 for all accessibility issues to be remediated

**Any other information?**

**Comment by thatblindgeye** [comment]
FYI for the PF team: This should be updated in both v5 and v6. An issue was opened for v5 in #10350
